### ISSUE TITLE
Pytorch3d error

### ISSUE THREAD

**Issue by kakao014** [philgras/neural-head-avatars#7]
Thanks for your work.

But i got install error:
when i install the requires by your setup.py:  the pytorch3d-0.6.1-cp39-cp39-linux_x86_64.whl is no longer accessable.

<Code>AccessDenied</Code>
<Message>Access Denied</Message>

And then i manually installed the pytorch3d,  it will be the version of 0.6.2, and i cannot find the 0.6.1 linux version online.
So after that, i run the notebook on the Colab, it would be many errors about pytorch3d...

If there is some solutions , very thanks.

**Comment by malteprinzler** [comment]
You are right, unfortunately, the prebuilt linux wheels for pytorch3d are only available for pytorchv 1.11.0 from now on (see https://github.com/facebookresearch/pytorch3d/blob/main/INSTALL.md#3-install-wheels-for-linux). 
Consequently, you'll have to install pytorch 1.11.0 following the instructions here https://pytorch.org/get-started/locally/ 
and then install pytorch3d via 
```pip install --no-index --force-reinstall --no-cache-dir pytorch3d -f https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu113_pyt1110/download.html```

Hope this helps. Please reach out if you face any more issues.

**Comment by kakao014** [comment]
Thanks a lot, i'll try it.

**Comment by juancprzs** [comment]
Hi there, @kakao014. I have been unable to install these dependencies.

Could you please confirm this solution worked for you?
If it did, could you share what, exactly, did you do?

Thanks!

**Comment by ohjarwa** [comment]
Hi @juancprzs  @kakao014  I met the same question.

Could you please share your solution if you have made it?

Thank you!

**Comment by edhyah** [comment]
Did not work for me - when downloading pytorch3d, I get the error:
```
Collecting pytorch3d
  Downloading https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu113_pyt1110/pytorch3d-0.7.2-cp39-cp39-linux_x86_64.whl (19.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 19.6/19.6 MB 18.1 MB/s eta 0:00:00
ERROR: Could not find a version that satisfies the requirement iopath (from pytorch3d) (from versions: none)
ERROR: No matching distribution found for iopath
```

**Comment by tobias-kirschstein** [comment]
I solved this by replacing the dependencies in both `setup.py` and `deps/video-head-tracker/setup.py` with:
```
"torch@https://download.pytorch.org/whl/cu113/torch-1.11.0%2Bcu113-cp39-cp39-linux_x86_64.whl",
"torchvision@https://download.pytorch.org/whl/cu113/torchvision-0.12.0%2Bcu113-cp39-cp39-linux_x86_64.whl",
"pytorch3d@https://dl.fbaipublicfiles.com/pytorch3d/packaging/wheels/py39_cu113_pyt1110/pytorch3d-0.7.2-cp39-cp39-linux_x86_64.whl",
```
